### PR TITLE
meom-ige: add dakkar-demo, a temporary basehub

### LINKED ISSUE #2049: [Request deployment] New Hub: demo-drakkar
### Important dates

- **Target start date**:  2023-01-23
- **Required start date**:  2023-01-27
- **Any important dates for usage**:  2023-01-30

### Hub Authentication Type

GitHub (e.g., @mygithubhandle)

### First Hub Administrators

Aurélie Albert, aurelie.albert@univ-grenoble-alpes.fr, @auraoupa
Takaya Uchida, takaya.uchida@univ-grenoble-alpes.fr, @roxyboy

### [GitHub Auth only] How would you like to manage your users?

Manually, by adding specific GitHub handles in the JupyterHub Admin panel

### [GitHub Teams Auth only] Profile restriction based on team membership

_No response_

### Hub logo image URL

https://drakkar2023.sciencesconf.org/data/header/DrakkarOcean.png

### Hub logo website URL

https://drakkar2023.sciencesconf.org/

### Hub user image GitHub repository

https://github.com/auraoupa/hub-user-image-drakkar-demo

### Hub user image tag and name

quay.io/auraoupa/drakkar-demo:62fa0b23aca5

### Extra features you'd like to enable

- [ ] Specific cloud provider or datacenter (otherwise GCP)
- [ ] Dedicated Kubernetes cluster
- [ ] Scalable Dask Cluster

### (Optional) Preferred cloud provider

GCP (preferred)

### (Optional) Billing and Cloud account

None

### Other relevant information to the features above

_No response_

### Tasks to deploy the hub

- [x] 1. Deploy information filled in above
- [x] 2. Engineer who will deploy the hub is assigned
- [x] 3. If using GitHub Orgs/Teams Auth, Engineer is given Owner rights to the org to set this up.
- [ ] 4. Initial Hub deployment PR <link to PR>
- [ ] 5. Administrators able to log on -> Hub now in steady-state

### PR DESCRIPTION
**Pull request by consideRatio**
- Closes #2049

### CODE REVIEW

File: config/clusters/meom-ige/cluster.yaml [hunk 1]
```diff
@@ -38,3 +38,13 @@ hubs:
...
       - common.values.yaml
       - prod.values.yaml
+  - name: drakkar-demo
+    display_name: "2023 DRAKKAR Ocean Modelling Workshop"
+    domain: drakkar-demo.meom-ige.2i2c.cloud
+    helm_chart: basehub
+    auth0:
+      # connection update? Also ensure the basehub Helm chart is provided a
+      # matching value for jupyterhub.custom.2i2c.add_staff_user_ids_of_type!
+      connection: github
```
Comment (GeorgianaElena) [inline config/clusters/meom-ige/cluster.yaml:48]: @consideRatio, I believe we generally wish to move away from using Auth0. Though, I don't think there's a clear timeline about when/if to remove it from our infrastructure.

For now, I believe you could use the GitHubOAuthenticator directly.

Comment (consideRatio) [reply config/clusters/meom-ige/cluster.yaml]: I think this is addressed by 00a928d

Comment (GeorgianaElena) [reply config/clusters/meom-ige/cluster.yaml]: Great! Thank you @consideRatio

**Review by GeorgianaElena — APPROVED** [review]
LGTM 🚀

**Comment by consideRatio** [comment]
Thank you for helping out with review @GeorgianaElena!!!

**Pull request merged**